### DISCUSSION TITLE
FFMPEG Not Available

### DISCUSSION THREAD

**Discussion by Thinline-Dynamic-Solutions** [chuot/rdio-scanner#323] [Q&A]
Every time my server starts it tells me that the ffmpeg is not available, no audio conversion will be performed. 

I'm not sure if there is anything i need to do.

It works fine however sometimes I get buzzing and no audio, I'm not sure if I'm missing any radio traffic.

Running Windows 11 Pro with the latest updates
Rdio-Scanner Latest Release
i5-9500 @3GHz
16 GB DDR4 2666MHz

**Comment by chuot** [comment]
You need to install FFMPEG on your PC. Please see the FAQ section of the PDF document provided with Rdio Scanner. There are some special steps on Windows.

FFMPEG will help normalize audio files so they play well on mobile devices.

**Comment by Thinline-Dynamic-Solutions** [comment]
I was able to get that figured out. I have a few more issues, but I believe
they are more related to SDR Trunk which I can figure out.

My newest issue is sometimes the database or application gets glitched
after doing configuration changes to where the server becomes completely
inaccessible on the listening side. I have to rebuild from scratch to get
things going again. Any insight or troubleshooting I could try on this
would be greatly appreciated. If I need to change the operating system to
dragon os or anything I’m willing to do that.
